### PR TITLE
Add installation guide

### PR DESCRIPTION
**Pull request by nahueespinosa**
### Proposed changes

Related to #165.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [ ] 🚀 Feature (change which adds functionality)
- [x] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [x] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### CODE REVIEW

File: INSTALLING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+   ```bash
+   cd ~/ws
+   git clone https://github.com/Ekumen-OS/beluga.git ~/ws/src/beluga
+   ```
+
+1. **Source the ROS installation**.
+
+   ```bash
+   . /opt/ros/${ROS_DISTRO}/setup.sh
```
Comment (hidmic) [inline INSTALLING.md:21]: @nahueespinosa nit^N:
```suggestion
   source /opt/ros/${ROS_DISTRO}/setup.bash
```
for consistency elsewhere.

File: INSTALLING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+1. **Install dependencies**.
+
+   ```bash
+   rosdep install -r --from-paths ~/ws/src/beluga -y -i -t build -t exec --skip-keys 'flatland_server flatland_plugins'
+   ```
+
+1. **Build and source the workspace**.
+
+   In ROS 2 and ROS 1 distributions (if colcon is installed), run:
```
Comment (hidmic) [inline INSTALLING.md:32]: @nahueespinosa nit: 
```suggestion
   In ROS 2 and ROS 1 distributions (if [`colcon`](https://colcon.readthedocs.io/en/released/user/installation.html) is installed), run:
```

**Review by hidmic — DISMISSED** [review]
LGTM barring minor comments.

File: INSTALLING.md (hunk 2 above)
Comment (nahueespinosa) [reply INSTALLING.md]: Nice suggestion! Thanks!

File: INSTALLING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+   Run `rviz` or `rviz2` depending on your distribution.
+
+   In ROS 2, when subscribing to the output topics from AMCL, we recommend the following [QoS](https://docs.ros.org/en/rolling/Concepts/Intermediate/About-Quality-of-Service-Settings.html) settings:
+
+   | Topic            | Depth | History      | Reliability  | Durability      |
+   |------------------|-------|--------------|--------------|-----------------|
+   | `particle_cloud` | 5     | Keep last    | Best effort  | Volatile        |
+   | `map`            | 5     | Keep last    | Reliable     | Transient local |
+   | `pose`           | 5     | Keep last    | Reliable     | Volatile        |
```
Comment (francocipollone) [inline INSTALLING.md:68]: :rocket:

Comment (hidmic) [reply INSTALLING.md:68]: We may as well provide a sample `.rviz` display config file too. No need to block this PR on it though.

**Review by hidmic — APPROVED** [review]

File: INSTALLING.md (hunk 3 above)
Comment (nahueespinosa) [reply INSTALLING.md]: Done in https://github.com/Ekumen-OS/beluga/pull/229/commits/bcba09b00f64a1e8515be012ac69fbcc349b0d4b.

**Pull request merged**